sprokit/processes/core/downsample_process.cxx

```diff
@@ -0,0 +1,239 @@
+/*ckwg +29
```

Comment:
downsample_process is perhaps not a great name for what I assume is a throttling of frame rate.